### PR TITLE
Update readme#44

### PR DESCRIPTION
**Pull request by sabin-thapa**
Added more information to the README:
1. CONTRIBUTING 
2. INSTALLATION
3. MIT LICENSE

### CODE REVIEW

**Comment by sabin-thapa** [comment]
@Rajkumar-justcoder 
I have made changes to the README and added an MIT LICENSE.
Please review and merge and also add the hacktoberfest-accepted label.
Thanks!
:clinking_glasses:

File: README.md [hunk 1]
```diff
@@ -1,24 +1,74 @@
...
+
+
+```bash
+git add <path to the file you worked on>
+```
+
+**3**. Commit your changes with
+
+```bash
+git cz
```
Comment (Rajkumar-justcoder) [inline README.md:50]: whats this git cz ?

Comment (Rajkumar-justcoder) [reply README.md:50]: ig its for commit msg then used do  preffered way

**Review by Rajkumar-justcoder — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sabin-thapa) [reply README.md]: Done :heavy_check_mark:  @Rajkumar-justcoder

**Comment by Rajkumar-justcoder** [comment]
Looks good to me

**Pull request merged**